examples/reference_environments/README.md

```diff
@@ -27,6 +27,12 @@ This example will show you how to start a reference environment for a postgres d
 
 2. Click on the jupyter notebook link in the output to open the notebook in your browser.
 
+If there are more than one URL being displayed, you can use the following command to only show the URL for the notebook.
+
+    ```bash
+    great_expectations example postgres --url
```

Comment:
Explanation for `--url` flag was added to the base README.md

Reply:
Thank you!